src/bundle/Resources/public/js/scripts/fieldType/ezgmaplocation.js

```diff
@@ -618,5 +620,14 @@
         }
     });
 
+    $('.ez-tabs a').on('shown.bs.tab', (event) => {
+        const id = event.target.getAttribute('href');
+        const element = doc.querySelector(id);
+
+        if (element.querySelectorAll(SELECTOR_FIELD).length > 0) {
+            maps.forEach(map => map.invalidateSize(true));
```

Comment:
code formatting (Prettier)
```suggestion
            maps.forEach((map) => map.invalidateSize(true));
```